styles/globals.css

```diff
@@ -63,4 +63,12 @@
     margin-left: -0.665em;
     @apply text-accent-error;
   }
+
+  .flex-container{
```

Comment:
we should avoid adding custom classes, but use tailwind provided styling if it's provided
https://tailwindcss.com/docs/flex